core/lib/dal/src/accounts_dal.rs

```diff
@@ -42,15 +42,21 @@ impl AccountsDal<'_, '_> {
             })
             .collect();
         let rows = sqlx::query!(
-            r#"
-                SELECT storage.value as "value!",
-                    tokens.l1_address as "l1_address!", tokens.l2_address as "l2_address!",
-                    tokens.symbol as "symbol!", tokens.name as "name!", tokens.decimals as "decimals!", tokens.usd_price as "usd_price?"
-                    FROM storage
-                INNER JOIN tokens ON
-                    storage.address = tokens.l2_address OR (storage.address = $2 AND tokens.l2_address = $3)
-                WHERE storage.hashed_key = ANY($1) AND storage.value != $4
-            "#,
+            "SELECT storage.value AS \"value!\", \
+                    tokens.l1_address AS \"l1_address!\", \
+                    tokens.l2_address AS \"l2_address!\", \
+                    tokens.symbol AS \"symbol!\", \
+                    tokens.name AS \"name!\", \
+                    tokens.decimals AS \"decimals!\", \
+                    tokens.usd_price AS \"usd_price?\" \
+               FROM storage \
+              INNER JOIN tokens ON storage.address = tokens.l2_address \
+                 OR ( \
+                    storage.address = $2 \
+                AND tokens.l2_address = $3 \
+                    ) \
+              WHERE storage.hashed_key = ANY ($1) \
+                AND storage.value != $4",
```

Comment:
I suggest using [raw strings](https://tourofrust.com/63_en.html) you can get rid of the backslashes here.

Reply:
@montekki see discussion here: https://github.com/matter-labs/zksync-era/pull/498#discussion_r1395441556